assets/brunch-config.js

```diff
@@ -0,0 +1,70 @@
+const neat = require('bourbon-neat').includePaths;
+const bourbon = require('bourbon').includePaths;
+
+exports.config = {
+  // See http://brunch.io/#documentation for docs.
+  files: {
+    javascripts: {
+      joinTo: "js/app.js"
+
+      // To use a separate vendor.js bundle, specify two files path
+      // http://brunch.io/docs/config#-files-
+      // joinTo: {
+      //   "js/app.js": /^js/,
+      //   "js/vendor.js": /^(?!js)/
+      // }
+      //
+      // To change the order of concatenation of files, explicitly mention here
+      // order: {
+      //   before: [
+      //     "vendor/js/jquery-2.1.1.js",
+      //     "vendor/js/bootstrap.min.js"
+      //   ]
+      // }
+    },
+    stylesheets: {
+      joinTo: "css/app.css"
+    },
+    templates: {
+      joinTo: "js/app.js"
+    }
+  },
+
+  conventions: {
+    // This option sets where we should place non-css and non-js assets in.
+    // By default, we set this to "/assets/static". Files in this directory
+    // will be copied to `paths.public`, which is "priv/static" by default.
+    assets: /^(static)/
+  },
+
+  // Phoenix paths configuration
+  paths: {
+    // Dependencies and current project directories to watch
+    watched: ["static", "css", "js", "vendor"],
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)